src/main/java/chess/Board.java

```diff
@@ -0,0 +1,22 @@
+package chess;
+
+import pieces.Pawn;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+    private List<Pawn> board = new ArrayList<>();
```

Comment:
pawn list 멤버 변수 이름으로 board 는 맞지 않는 것 같습니다. 이미 Board 클래스의 인스턴스에도 board라는 이름을 쓸 것인데 (테스트코드를 보면..) 

```suggestion
    private List<Pawn> pawns = new ArrayList<>();
```
로 수정해주세요. ㅎㅎ

Reply:
네! 제가 해당 미션의 내용을 잘못이해하여 멤버 변수 명을 잘못 지은 것 같습니다.  말씀 주신 사항으로 수정했습니다!